docs/01_cva6_user/Traps_Interrupts_Exceptions.rst

```diff
@@ -1,25 +1,126 @@
-﻿..
-   Copyright (c) 2023 OpenHW Group
-   Copyright (c) 2023 Thales DIS design services SAS
-
-   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
-
-.. Level 1
-   =======
-
-   Level 2
-   -------
-
-   Level 3
-   ~~~~~~~
-
-   Level 4
-   ^^^^^^^
-
-.. _cva6_traps_interrupts_exceptions:
-
-Traps, Interrupts, Exceptions
-=============================
-Gap for step1 verification. Reuse DVplan from E4? Jean-Roch will check with André. Mike will reach out to some people who could help.
-We expect this section to be 1 page.
-
+﻿..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales DIS SAS
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_traps_interrupts_exceptions:
+
+Traps, Interrupts, Exceptions
+=============================
+Traps are composed of interrupts and exceptions.
+Interrupts are asynchronous events whereas exceptions are synchronous ones.
+On one hand, interrupts are occuring independently of the instructions
+(mainly raised by peripherals or debug module).
+On the other hand, an instruction may raise exceptions synchronously.
+
```

Comment:
Suggest to mention that U-mode delegation of interrupts and exceptions is not supported.

Reply:
AFAIK, RISC-V privilege spec does not support (even optionally) U-mode delegation. So, I'm not sure it is useful to mention that in our manual.